mesa/batchrunner.py

```diff
@@ -102,7 +102,7 @@ def batch_run(
                     run_id = next(run_counter)
                     data = []
                     for run_data in rawdata:
-                        out = {"RunId": run_id, "iteration": iteration - 1}
+                        out = {"RunId": run_id, "iteration": iteration + 1}
```

Comment:
Does this `iteration` for the mp version really start from 0?

Reply:
Yes, it's defined as a range: https://github.com/projectmesa/mesa/blob/f390714fa4eae5c6c8956a00b377db6a376ce88d/mesa/batchrunner.py#L84

Reply:
That's true only when the `number_processes` is 1. Otherwise, it is an element from a `Counter` object.

Reply:
That's implemented a bit weird